internal/kubernetes/operator/version/provider.go

```diff
@@ -25,7 +25,8 @@ import (
 )
 
 const (
-	operatorRepositoryOrg     = "mongodb"
+	operatorRepositoryOrg = "mongodb"
+	//nolint:gosec
 	operatorRepository        = "mongodb-atlas-kubernetes"
```

Comment:
Apparently this string looks like leaked credentials :)

Reply:
nit: I prefer the inline style, I think it's clearer to tell which line was affected
```suggestion
	operatorRepository        = "mongodb-atlas-kubernetes" //nolint:gosec // false positive
```